definitions/tools/kmer_size_from_index.cwl

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "Helper script to pull the k-mer size used in generating a kallisto index from the index file"
+baseCommand: ["g++", "-o", "extractor", "extractor.cpp"]
```

Comment:
This seems clever to be able to compile C++ on the fly, but should it be  compiled once into a docker image once instead of every time a workflow runs?

Reply:
Yeah, this is a lot easier to read. Updating